### PR TITLE
add precourse files

### PR DESCRIPTION
**Pull request by xavi016**

### CODE REVIEW

File: staff/xavier-navarro/precourse/airlines-pro/index.js [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    }
+    
+    if(result.length>1){
+        let purchase
+        console.log("Se ha encontrada más de un vuelo con es precio. ¿Cuál desea coger?")
+        let idFlights = []
+        result.forEach(elem => {
+            idFlights.push(elem['id'])
+            console.log("- Vuelo con ID "+elem['id']+" con origen "+elem['to']+" y destino "+elem['to'])
+        });
```
Comment (manuelbarzi) [inline staff/xavier-navarro/precourse/airlines-pro/index.js:201]: WTF is this semicolon doing here when almost all your code has no one?

File: staff/xavier-navarro/precourse/airlines-pro/index.js [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+                case "=":
+                    flight = getEqualPrice()
+                    if(flight === false){
+                        console.log("No se ha encontrado un vuelo con ese precio.")
+                        console.log("Hasta la próxima.")
+                    }else {
+                        console.log("Ha elegido el vuelo "+flight['id']+ " con destino "+flight['from']+". Precio: "+flight['cost'])
+                        console.log("Gracias por su compra, vuelva pronto.")
+                    }
+                    break;
```
Comment (manuelbarzi) [inline staff/xavier-navarro/precourse/airlines-pro/index.js]: semicolon now? no way!

File: staff/xavier-navarro/precourse/airlines/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    console.log(welcome()+", estos són todos los vuelos disponibles:")
+    printFlights(flights)
+    console.log("El coste medio de los vuelos es de "+priceAverage() )
+    console.log("Vuelos con escala:")
+    scaleFlights()
+    console.log("Los últimos 5 vuelos del día són:")
+    console.log(lastFlights(5))
+}
+
+SkylabAirlines()
```
Comment (manuelbarzi) [inline staff/xavier-navarro/precourse/airlines/index.js:68]: functions in js follow the naming convention of camelCase. update it.

File: staff/xavier-navarro/precourse/bingo/index.js [hunk 4]
```diff
@@ -0,0 +1,218 @@
+function playerName(){
+    let user = prompt("Introduce your user, please")
+    return user
+}
+function generateBingoCard(totalCardLines){
+    const totalNumCard = 15
```
Comment (OscarErre) [inline staff/xavier-navarro/precourse/bingo/index.js:6]: molaria que el jugador pudiera elegir el tamaño del cartón :P

**Comment by xavi016** [comment]
@manuelbarzi  errors fixed

File: staff/xavier-navarro/precourse/pasapalabra-2/index.js [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+  let playerScore
+  let recordQuestSelected = []
+  let questionFound = false
+  // Current position of the letter
+  let currentLetter = 0
+  let firstLapDone = false
+  let isComplete = false
+  let roscoTimer
+  let score = 0
+  const totalLetters = 27
```
Comment (OscarErre) [inline staff/xavier-navarro/precourse/pasapalabra-2/index.js]: aquí se podría usar el totalLetters = questions.length?

Comment (xavi016) [reply staff/xavier-navarro/precourse/pasapalabra-2/index.js]: Tienes razón! Cambiado

**Pull request merged**